deploy/kustomize/overlays/cloudbeat-eks/config-map.yml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudbeat-daemonset-config
+  namespace: kube-system
+  labels:
+    k8s-app: cloudbeat
+data:
+  cloudbeat.yml: |-
+    cloudbeat:
```

Comment:
consider using configmap generator and taking the config.yaml to file

Reply:
@uri-weisman I am pretty sure that I remove it when I made the last Kustomize refactor since it wasn't needed anymore.

If you wish to do this change, I guess you'll find out how to do it there.